docs/Building.md

```diff
@@ -2,6 +2,10 @@
 
 We use **Tauri** which requires **_[some extra setup]_**.
 
+Be sure to clone this repo with **submodules:**
+  * `git clone --recursive https://github.com/LucidVR/opengloves-ui.git` 
```

Comment:
I believe `--recursive` is an older flag, can you switch it to `--recurse-submodules` please?